assets/javascripts/discourse-akismet/connectors/topic-above-post-stream/topic-removed-notification.gjs

```diff
@@ -0,0 +1,33 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { service } from "@ember/service";
+import helperFn from "discourse/helpers/helper-fn";
+import { i18n } from "discourse-i18n";
+
+const DELETED_CHANNEL_PREFIX = "/discourse-akismet/topic-deleted/";
+
+export default class TopicRemovedNotification extends Component {
+  @service messageBus;
+
+  @tracked akismetFlaggedTopic;
+
+  subscribeMessageBus = helperFn(({ topicId }, on) => {
+    this.akismetFlaggedTopic = false;
```

Comment:
I think this can be part of the tracked prop definition?

Reply:
It needs to reset to `false` whenever the topicID changes (e.g. when you navigate between two routes)